gateleen-routing/src/main/java/org/swisspush/gateleen/routing/auth/AuthHeader.java

```diff
@@ -0,0 +1,36 @@
+package org.swisspush.gateleen.routing.auth;
+
+public class AuthHeader {
+    private final String key = "Authorization";
+    private final String value;
```

Comment:
probably not relevant for us here, but basically HTTP headers allow multiple values (per key)

Reply:
You are right. I also think it is not relevant here. I don't see a case where we would have multiple tokens.